tests/Dfe.EarlyYearsQualification.UnitTests/Helpers/PlaceholderUpdaterTests.cs

```diff
@@ -0,0 +1,43 @@
+using Dfe.EarlyYearsQualification.Web.Helpers;
+using FluentAssertions;
+
+namespace Dfe.EarlyYearsQualification.UnitTests.Helpers;
+
+[TestClass]
+public class PlaceholderUpdaterTests
```

Comment:
Suggestion (minor): might be worth a test to check that if the text contains multiple copies of the placeholder then all of them are replaced? `"The year $[actual-year] is $[actual-year]$"`. 🙂

Reply:
Added that now :-)